compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -28,15 +28,12 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None):
 
   import Rendering._
 
-  private val MaxStringElements: Int = 1000  // no need to mkString billions of elements
-
   private var myClassLoader: AbstractFileClassLoader = _
 
   private var myReplStringOf: Object => String = _
 
-
   /** Class loader used to load compiled code */
-  private[repl] def classLoader()(using Context) =
+  private[repl] def classLoader()(using ctx: Context) =
```

Comment:
This change is not necessary. `ctx` is already available as a method summoning `Context`

Reply:
very true - dropped